data.sql

```diff
@@ -34,3 +34,46 @@ VALUES
 UPDATE animals SET Species = 'digimon' WHERE Name like '%mon';
 
 UPDATE animals SET Species = 'pokemon' WHERE Species = 'unspecified';
+
+INSERT INTO owners
+  (full_name, age)
+VALUES
+  ('Sam Smith ', 34),
+  ('Jennifer Orwell', 19 ),
+  ('Bob', 45 ),
+  ('Melody Pond', 77 ),
+  ('Dean Winchester', 14 ),
+  ('Jodie Whittaker', 38 );
```

Comment:
![Screenshot from 2022-09-14 17-44-59 (copy)](https://user-images.githubusercontent.com/93039359/190157584-52fd4bbf-dde2-4e90-b2aa-91c8c0022d80.png)

- [ ] Your are not adding an id for each entry and id cannot be null so that's why it showing the error which has been showed in the above image :arrow_up:

Reply:
@HAMMAS-SALEEM  Hi, I don't understand what do you want me to do. Because I have ID for owners 
![ID of owners](https://user-images.githubusercontent.com/88398101/190166524-334e59a9-1525-4b44-b9d0-d7aa58f8af3e.png)
And also have the column (name) in table spe
![Column name in species](https://user-images.githubusercontent.com/88398101/190166735-bfd2c1c1-abaa-4f39-81f6-49f3b5217d19.png)
cies.

Reply:
Dear @digia3d You need to give id in parameters because id cannot be null. That's what i want you to do here.